alg/teca_2d_component_area.h

```diff
@@ -57,18 +73,23 @@ class teca_2d_component_area : public teca_algorithm
     TECA_GET_ALGORITHM_PROPERTIES_DESCRIPTION()
     TECA_SET_ALGORITHM_PROPERTIES()
 
-    // set the name of the input array
+    // The component variable name containing the region labels.
```

Comment:
Use doxygen style comments and grouping here as well.

The documentation should read something to the effect:
```
Set/get the name of the variable  containing the connected component labels
```